packages/devp2p/README.md

```diff
@@ -61,22 +61,37 @@ includes node discovery ([./src/dpt/server.ts](./src/dpt/server.ts))
 Create your peer table:
 
 ```ts
-import { DPT } from '@ethereumjs/devp2p'
-import { hexToBytes } from '@ethereumjs/util'
+// examples/dpt.ts
 
-const dpt = new DPT(hexToBytes(PRIVATE_KEY), {
-  endpoint: {
-    address: '0.0.0.0',
-    udpPort: null,
-    tcpPort: null,
-  },
-})
+import { DPT } from '@ethereumjs/devp2p'
+import { bytesToHex, hexToBytes, randomBytes } from '@ethereumjs/util'
+
+const PRIVATE_KEY = hexToBytes('0xed6df2d4b7e82d105538e4a1279925a16a84e772243e80a561e1b201f2e78220')
+const main = async () => {
+  const dpt = new DPT(PRIVATE_KEY, {
+    endpoint: {
+      address: '0.0.0.0',
+      udpPort: null,
+      tcpPort: null,
+    },
+  })
+  console.log(`DPT is active and has id - ${bytesToHex(dpt.id!)}`)
+  // 0xcd80bb7a768432302d267729c15da61d172373ea03679f3c8156b55453b68668a482d183a27139d7f37cc0c83eb7f414a74d6256d0b2bb20655041b9a3c8e0ed
```

Comment:
A bit of a nit but being explicit that this comment is the logged output of the above statement or just moving the comment to the same line since it still flows off the screen when this is rendered as markdown can potentially clear up some confusion about what the hash is.

Reply:
Fair, I've updated this to truncate the logging and hopefully clarify what's going on.